spec/imsc-hrm.html

```diff
@@ -11,16 +11,16 @@
     }
   </style>
 <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
-</script>
   <script class='remove'>
 var respecConfig = {
-                    specStatus:   "WD"
+                    specStatus:   "CR"
                 //,   processVersion: "2018"
                 //,   previousMaturity: "PR"
                 //,   publishDate: "2021-11-09"
                 //,   modificationDate: "2020-04-27"
                 //,   errata: ""
-                //,   implementationReportURI: ""
+                ,   crEnd: "2023-03-01"
```

Comment:
Obviously this date will need to change, suggest publication date + 8 weeks (4 is the process-defined minimum, I believe).